automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSpec.scala

```diff
@@ -333,6 +336,33 @@ class LeonardoSpec extends FreeSpec with Matchers with Eventually with ParallelT
       }
     }
 
+    "should localize files" in withWebDriver { implicit driver =>
+      implicit val token = ronAuthToken
+      val file = ResourceFile("diff-tests/import-hail.ipynb")
+
+      withNewCluster(project) { cluster =>
+        withFileUpload(cluster, file) { _ =>
+          //good data
+          val goodLocalize = Map(
+            "test.rtf" -> s"$swatTestBucket/test.rtf"
+            //TODO: create a bucket and upload to there
+            //"gs://new_bucket/import-hail.ipynb" -> "import-hail.ipynb"
+          )
+          Leonardo.notebooks.localize(cluster.googleProject, cluster.clusterName, goodLocalize)
+          val localizationLog = Leonardo.notebooks.getContentItem(cluster.googleProject, cluster.clusterName, "localization.log")
+          localizationLog.content shouldBe defined
+          localizationLog.content.get shouldNot include("Exception")
+
```

Comment:
Should we also check that `test.rtf` is on the notebook server? Could do something like:
```
notebookPage.executeCell("! ls ~/test.rtf") shouldBe Some("/home/jupyter-user/test.rtf")
```

Reply:
oh right, yes! i can also do `getContentItem` like below